assets/src/components/activities/adaptive/components/authoring/LayoutEditor.tsx

```diff
@@ -220,14 +251,14 @@ const LayoutEditor: React.FC<LayoutEditorProps> = (props) => {
 
   const handlePartDelete = useCallback(async () => {
     // console.log('AUTHOR PART DELETE', { selectedPart }, selectedPartId);
-    const filteredParts = parts.filter((part) => part.id !== selectedPart.id);
+    if (!selectedPartAndCapabilities) return;
```

Comment:
After adding some typing to the capabilities part, there was a potential-null compiler warning that popped up, that's the reason for this early return.